### PR TITLE
Feature/us1263 migrate mp data

### PR DESCRIPTION
**Pull request by IngageKrizJ**
Add migration process for moving MP data from integration to demo

### CODE REVIEW

File: MigrateMPData/MigrateMPData/Program.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Configuration;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (IngageMaddoxT) [inline MigrateMPData/MigrateMPData/Program.cs]: Some of these using are not needed.

File: MigrateMPData/MigrateMPData/MinistryPlatformTableConfigReader.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+using System.IO;
+using System.Reflection;
+
+namespace MigrateMPData
+{
+    public class MinistryPlatformTableConfigReader : IMinistryPlatformTableConfigReader
+    {
+        private static readonly ILog logger = LogManager.GetLogger(MethodBase.GetCurrentMethod().DeclaringType);
+
+        public MinistryPlatformTableConfigReader()
```
Comment (IngageMaddoxT) [inline MigrateMPData/MigrateMPData/MinistryPlatformTableConfigReader.cs]: empty constructor?

File: MigrateMPData/MigrateMPData/Program.cs (hunk 1 above)
Comment (IngageKrizJ) [inline MigrateMPData/MigrateMPData/Program.cs]: I went through all .cs files and remove unused and sorted all usings (I'm sure resharper has a way to do that in bulk...)

File: MigrateMPData/MigrateMPData/MinistryPlatformTableConfigReader.cs (hunk 2 above)
Comment (IngageKrizJ) [inline MigrateMPData/MigrateMPData/MinistryPlatformTableConfigReader.cs]: Removed - that was left over from when I had a constructor that took in a filename, removed that from the constructor, but left the empty constructor...

**Pull request merged**